bootstrap/api/v1alpha1/openshiftassistedconfig_types.go

```diff
@@ -23,8 +23,7 @@ import (
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+const DiscoveryIgnitionOverrideAnnotation = "openshiftassistedconfig.cluster.x-k8s.io/discovery-ignition-override"
```

Comment:
this will be replaced by proper API fields in the next release https://issues.redhat.com/browse/MGMT-20824